README.md

```diff
@@ -0,0 +1,4 @@
+# virtual-puppet-project-website
+
+The source code for the Virtual Puppet Project (VPupPr) website.</br>
```

Comment:
Avoid using a raw \<br> tag here. Markdown supports newlines if you add an extra newline between sentences.